docs/flow/modin/experimental/index.rst

```diff
@@ -15,7 +15,7 @@ and provides a limited set of functionality:
 * :doc:`xgboost <xgboost>`
 * :doc:`sklearn <sklearn>`
 * :doc:`batch <batch>`
-* :doc:`Range-partitioning GroupBy implementation <range_partitioning_groupby>`
+* :doc:`Range-partitioning implementations <range_partitioning_groupby>`
```

Comment:
```suggestion
* :doc:`Range-partitioning implementations: <range_partitioning_groupby>`
```

Reply:
don't think a colon symbol is required here, it's just a list of links:
https://modin.readthedocs.io/en/stable/flow/modin/experimental/index.html

Reply:
You have plural implementations, how are you planning to specify multiple functions/files?

Reply:
I'm planning to rename `range_partitioning_groupby.rst -> range_partitioning.rst` and use this file to store information about all the methods that have range-partitioning implementation, then I'm planing to move that page to the [optimization notes](https://modin.readthedocs.io/en/stable/usage_guide/optimization_notes/index.html) (I want to do all this in a separate PR)

Reply:
ok